### PR TITLE
(1/10) RUM-2895 create SR Compose module

### PR DESCRIPTION
**Pull request by xgouchet**
_This PR has been created automatically by splitting a large branch_

RUM-2895 create SR Compose module

### CODE REVIEW

File: features/dd-sdk-android-session-replay-compose/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Datadog Session Replay Extension Support for Material UI Components
```
Comment (0xnm) [inline features/dd-sdk-android-session-replay-compose/README.md:1]: it should be `Compose` here

File: features/dd-sdk-android-session-replay-compose/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+        // Required because AGP doesn't support kotlin test fixtures :/
+        java.srcDirs("${project.rootDir.path}/dd-sdk-android-core/src/testFixtures/kotlin")
+    }
+}
+
+dependencies {
+    api(project(":features:dd-sdk-android-session-replay"))
+    implementation(libs.kotlin)
+    implementation(libs.gson)
+    implementation(libs.googleMaterial)
```
Comment (0xnm) [inline features/dd-sdk-android-session-replay-compose/build.gradle.kts:58]: is this one needed?

File: features/dd-sdk-android-session-replay-compose/ComposeInternals.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
+
+
+## ComposeView & AndroidComposeView
```
Comment (0xnm) [inline features/dd-sdk-android-session-replay-compose/ComposeInternals.md:3]:
```suggestion
## ComposeView & AndroidComposeView
```

**Review by 0xnm — DISMISSED** [review]
lgtm! left some comments

**Review by hestonhoffman — DISMISSED** [review]
LGTM once other comments are addressed.

**Review by mariusc83 — APPROVED** [review]

**Review by jonathanmos — APPROVED** [review]

**Pull request merged**